crawl-ref/source/dat/des/branches/crypt.des

```diff
@@ -1463,6 +1486,884 @@ cG.MGM.GcG.1.GcG.1.GcG.1.Gc
+xDDDEEEFFFx
+...........
+xHHHJJJKKKx
+x.........x
+x0x0x0x0x0x
+xxxxxxxxxxx
+ENDMAP
+
+NAME:    nicolae_crypt_tutankhamun
+TAGS:    transparent
+DEPTH:   Crypt:2-
+KMONS:   D = guardian mummy / skeletal warrior / ancient champion / eidolon
+KMONS:   M = mummy priest / lich / ancient lich w:2 / dread lich w:2 / \
+             royal mummy w:2
+KMONS:   S = ushabti
+KMONS:   _ = mummy priest
+KFEAT:   _ = altar_kikubaaqudgha / altar_yredelemnul
+SUBST:   D = D., * = ***|, { = {[(
+NSUBST:  Z = S / SDD / D
+MAP
+xxxxxxxxxxxxxxxxxx
+xx.......+..%*xxxx
+xx..GZG..x.D%*xxxx
+xx..ZMZ..xDS%_xxxx
+xx..GZG..x.D%*xxxx
+xx.......x..%*xxxx
+xxxxx....xxxxxxxxx
+xxxxx....xxxxxxxxx
+xxxxx....xxxxxxxxx
+xxxxx....xG.....Gx
+x%D.x....+...{...+
+x%D.x....xG.....Gx
+x%D.+....xxxxxxxxx
+x%D.x....xxxxxxxxx
+xxxxxxxxxxxxxxxxxx
+ENDMAP
+
+NAME:  nicolae_crypt_wrong_turn
+TAGS:  transparent
+DEPTH: Crypt
+SUBST: V : VGx...
+MAP
+xxxxxxxxxxxxxxx@@x
+xGGxxxx....xxxx..x
+x..xxxV....Vxxx..x
+x..xx........xx..x
+x.......88.......x
+x.......88.......x
+x..xx........xx..x
+x..xxxV....Vxxx..x
+x..xxxx....xxxxGGx
+x@@xxxxxxxxxxxxxxx
+ENDMAP
+
+NAME:    nicolae_crypt_vampire_mansion
+TAGS:    transparent
+DEPTH:   Crypt
+KITEM:   a = any weapon good_item / any armour good_item
+KITEM:   d = any book / any scroll / nothing w:5
+KITEM:   e = any potion good_item
+KITEM:   f = any jewellery good_item / any wand good_item w:5 / \
+             any magical staff good_item w:3
+KITEM:   ' = any randbook
+KMONS:   d! = shadow wraith / freezing wraith / flayed ghost / eidolon / \
+              nothing w:40
+KMONS:   pq = vampire knight
+KMONS:   rs = vampire mage
+KMONS:   z = ghoul
+KMONS:   R = vampire / jiangshi / ghoul
+KMONS:   LY = lich / ancient lich w:5 / dread lich w:5
+KMONS:   Q = curse skull
+KFEAT:   - = iron_grate
+KFEAT:   _ = altar_yredelemnul / altar_kikubaaqudgha / altar_makhleb / \
+             altar_dithmenos
+KFEAT:   Y = Y
+SHUFFLE: pqrs, JK
+SUBST:   J = ., K = G, * = *., R = RRRR!!!!.
+NSUBST:  D = + / x, E = + / x, F = +x / +, H = +x / +, p = p / z / 2=a / .
+NSUBST:  q = q / a* / e% / x. / ., r = r / 2=x / %, s = s / Q / _ / f / f'
+NSUBST:  Z = L / 3=x / 3=x. / |
+MAP
+xxxxxxxxxxxxxxxxxxxxx
+xeeeD.x|xR+R....Eeeex
+xxDxx.....-.ZZZ.xxExx
+x*.*x..Y..+.ZZZ.x*.*x
+x.R.F.....-.ZZZ.H.R.x
+x*.*xRx|x.+....Rx...x
+xxFxxxxxx+x+xxxxx*.*x
+xJ.Kxp.px...xq.qxxHxx
+xKRJxp..+...+..qxd.dx
+xJ.Kxp.px...xq.qxx.xx
+xKRJxxxxx...xxxxxdRdx
+xJ.Kxs.sx...xr.rxx.xx
+xxFxxs..+...+..rxd.dx
+x...xs.sx...xr.rxxHxx
+x.R.xxxxx...xxxxx...x
+-...xG..G...G..Gx.R.-
+x...+...........+...x
+xx-xx.R.x.R.x.R.xx-xx
+ ...x...........x...
+ @..-...x...x...-..@
+    xxxxx+++xxxxx
+ENDMAP
+
+# You can break the angels out of prison, if you want.
+# I can't possibly recommend it.
+NAME:  nicolae_crypt_yred's_angel_processing
+TAGS:  transparent
+DEPTH: Crypt
+KITEM: H = | w:5 / nothing
+KMONS: K = death knight band
+KMONS: _ = patrolling profane servitor
+KMONS: H = cherub / angel / daeva / ophan / seraph
```

Comment:
No to placing seraphim here, their rarity and lateness of placement is rather diminished by spawning in a random Crypt vault compared to a single Pan level / zigs / the orb run. One could argue Yred's transforming of angels could make an ophanim have the servitor body plan, but I'd probably argue against those being here either, because just as many people will question that as they will find it a neat idea.